doc/dev/decisionrecords/Codestyle.md

```diff
@@ -58,7 +58,7 @@ Editor > Code Style_. Then select **Project** in the \_Scheme drop down.
 You can run the Prettier Maven plugin as an external tool in IntelliJ. Set it up as an
 `External tool` and assign a key-shortcut to the tool execution.
 
-![External Tool Dialog](../../../doc/user/images/ExternalToolDialog.png)
+![External Tool Dialog](../images/ExternalToolDialog.png)
```

Comment:
Btw why do we have one shared images folder instead of having one for decision records, for instance?